sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v2/query.sql

```diff
@@ -0,0 +1,404 @@
+CREATE TEMP FUNCTION normalize_browser(browser STRING) AS (
```

Comment:
I don't remember if I already asked this, would this UDF be useful anywhere outside of this query?